integration-cli/docker_cli_ps_test.go

```diff
@@ -509,8 +519,9 @@ func (s *DockerSuite) TestPsRightTagName(c *check.C) {
 }
 
 func (s *DockerSuite) TestPsLinkedWithNoTrunc(c *check.C) {
+	// E2E: Test assumes no other containers.
 	// Problematic on Windows as it doesn't support links as of Jan 2016
-	testRequires(c, DaemonIsLinux)
+	testRequires(c, DaemonIsLinux, NotE2E)
```

Comment:
This test can be fixed by using a label and filter, or by just checking that the one expected lines are in the output and ignoring the rest.